src/styles/base/base.css

```diff
@@ -6,8 +6,7 @@
 }
 
 :root {
-  --fullcalendar-event-1: #434884;
-  --fullcalendar-event-2: #5C62B2;
-  --fullcalendar-event-3: #6D73CB;
-  --fullcalendar-event-4: #848CF0;
+  --fullcalendar-event-1: #571F7C;
+  --fullcalendar-event-2: #38164E;
+  --fullcalendar-event-3: #EBE6F0;
 }
```

Comment:
Missing space at the end of the document:
You may install and set an Eslint config for the project.

Reply:
I'm adding this space.